### DISCUSSION TITLE
Code Coverage RTL with cocotb in QuestaSim

### DISCUSSION THREAD

**Discussion by AntonBabundin** [cocotb/cocotb#2712] [Q&A] [answered]
How to run the RTL code сoverage at QuestaSim with using cocotb ?

**Comment by ktbarrett** [comment, marked as answer]
If you are using the makefiles, pass the appropriate coverage flags to `COMPILE_ARGS` and `SIM_ARGS`. For example, `COMPILE_ARGS=+cover`, `SIM_ARGS=-coverage`.

**Comment by AntonBabundin** [comment]
@ktbarrett  But he issues only total assertion coverage. How to view coverage across all branches in code ?

**Reply by ktbarrett** [reply]
Check the Questa manual. That is outside the scope of cocotb.

**Reply by AntonBabundin** [reply]
@ktbarrett Sorry, understood the problem
